tcms/bugs/tests/test_models.py

```diff
@@ -32,7 +34,9 @@ def test_notify_assignee_on_bug_creation(self, send_mail):
             "pk": bug.pk,
             "summary": bug.summary,
         }
-        expected_body = render_to_string("email/post_bug_save/email.txt", {"bug": bug})
+        expected_body = render_to_string(
+            "email/post_bug_save/email_create.txt", {"bug": bug}
+        )
```

Comment:
Instead of duplicating the template you can extend it with a `last_comment` variable and pass appropriate value in both cases. It would be beneficial for someone to see the latest state + details on a bug even if that is the first time they see the notification.  If I'm not mistaken this is how Bugzilla notifications work.